pkg/commands/diff/command.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package diff
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"sort"
+
+	"golang.org/x/exp/maps"
+
+	"github.com/abcxyz/pkg/cli"
+	"github.com/abcxyz/pkg/logging"
+
+	"github.com/abcxyz/guardian/internal/version"
+	"github.com/abcxyz/guardian/pkg/flags"
+	"github.com/abcxyz/guardian/pkg/git"
+	"github.com/abcxyz/guardian/pkg/terraform"
+	"github.com/abcxyz/guardian/pkg/util"
+)
+
+var _ cli.Command = (*DiffCommand)(nil)
+
+// DetectIamDriftCommand is a subcommand for Guardian that enables detecting IAM drift.
```

Comment:
I think this comment might have been copy-pasted without being updated.

Reply:
Updated.